verticaltabs.js

```diff
@@ -617,6 +636,40 @@ VerticalTabs.prototype = {
     }
   },
 
+  resizeTabs: function () {
```

Comment:
We need to call resizeTabs in many places though. I'd rather have that logic in one place.